packages/web-components/src/player-component/player.class.ts

```diff
@@ -270,8 +271,10 @@ export class PlayerWrapper {
         try {
             if (url.indexOf('wss:/') != -1) {
```

Comment:
This was not your doing, but this should be if (url.startsWith('ws')) to be consistent with the other place we check for this, and to permit (and not disallow) ws:// URLs.

Reply:
fixed